src/apiCalls.js

```diff
@@ -1,7 +1,7 @@
 // @flow
-import fetch from 'isomorphic-fetch';
 import {formatQueryParams} from './utils';
 import {getUrlForContactPoint, getPublicApiUrl, getContactPoint} from './globals';
+import fetch from './fetch';
```

Comment:
Can we call this wrapped version of fetch something else like `quiqFetch` or something? It seems a little weird to have a method called `fetch` that isn't the actual fetch call.